### ISSUE TITLE
Compiling documentation for micro-sam 

### ISSUE THREAD

**Issue by anwai98** [computational-cell-analytics/micro-sam#501]
First draft for working on adding documentation for micro-sam. Here are a few points I remember (and merged a few here from discussions floating around)

- Jupyter Notebook for:
    - [x] Fine-tuning
    - [x] Automatic Mask Generation and Automatic Instance Segmentation
    - [x] Model evaluation for interactive and automatic segmentation
      - Not so high priority, we should then link to it [here](https://github.com/computational-cell-analytics/micro-sam/blob/master/doc/bioimageio/validation.md). 
- Document the library:
    - [x] Experiment folders (minor)
    - [x] Sub-modules (major)
- FAQ Section to address commonly known issues / questions:
    - [x] Suggested installation methods for: (installing stuff from source / installing stuff from conda-forge)
    - [x] Check image.sc
    - [x] Check GitHub issues
    - [x] Discuss with team and jot down the known issues and tips to take care of
    - [x] Add pointers to sample scripts for using the python library or command line
    - [x] Suggestions to create dataloaders (WIP)
- Add tutorial videos:
    - [x] Quickstart
    - [x] Annotator 2d
    - [x] Annotator 3d
    - [x] Annotator tracking
    - [x] Image series annotator

Feel free to add more TODOs here!

**Comment by anwai98** [comment]
- [ ] Test the training UI and document the features (+video?) (especially parameter naming)

**Comment by constantinpape** [comment]
Also relevant: #197 on parameter documentation, which has some relevant links.

**Comment by anwai98** [comment]
I'll merge the minor parts missing here - https://github.com/computational-cell-analytics/micro-sam/issues/566. Most of the things are completed here. Thanks!
